components/search/Cargo.toml

```diff
@@ -8,3 +8,5 @@ errors = { path = "../errors" }
 content = { path = "../content" }
 config = { path = "../config" }
 libs = { path = "../libs" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
use `libs::serde_json` instead so we only have the lib defined in one crate

Reply:
done; is there a similar way to get Serde?

Reply:
i don't think so because of the derive feature